### PR TITLE
Build holopy with windows support

### LINKED ISSUE #15: nose package is not being installed
**Issue:**
`nosetests` run outside of `conda` environment. Tests immediately fail because `python2.7` is being used. If `nose` is not installed at all, tests fail because `nosetests` binary is not found.

**Solution:**
Install `nose` `conda` package in the same environment. It's obvious it can be added to `meta.yaml`, however, I'm not sure to what section of `requirements` (`build` or `run`) and is it possible without pumping the package version so I haven't filed the PR and opened an issue instead.

The alternative solution is to add information about `nose` package to the docs (`dev_tutorial`) but the package size is only about 200 Kb so I think it's ok to install it for everyone.

### PR DESCRIPTION
**Pull request by ralex0**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

Following up on @barkls comments on #19, This PR has a cleaner commit history and is based on barkls/holopy-feedstock@34a8d5e

We can update the top of meta.yaml to point at a manoharan-lab/holopy branch after manoharan-lab/holopy#304 is merged or it is at least up to date with ralex0/holopy@c2c2496

Edit: Also, this fixes #15

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,55 +1,59 @@
...
     - python
-    - numpy 1.12.*
+    - pip
+    - {{ pin_compatible('numpy') }}
     - setuptools
-    - gcc
-    - toolchain3
-
+    - m2w64-toolchain  # [win]
+    - nose
```
Comment (barkls) [inline recipe/meta.yaml]: I believe this should be a run dependency, not a host dependency.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
     - python
-    - numpy >=1.12
+    - {{ pin_compatible('numpy') }}
     - scipy
     - pyyaml
     - pillow
     - h5py
-    - emcee
     - matplotlib
     - xarray >=0.10.0
```
Comment (barkls) [inline recipe/meta.yaml]: This is out of date. We can use any currently supported version of xarray.

**Comment by barkls** [comment]
Also fixes #14

File: recipe/meta.yaml (hunk 2 above)
Comment (ralex0) [reply recipe/meta.yaml]: changed in @19dce31

File: recipe/meta.yaml (hunk 1 above)
Comment (ralex0) [reply recipe/meta.yaml]: changed in @1b122e9

**Comment by barkls** [comment]
Extended in #24

**Pull request closed without merging**